js/row-actions.js

```diff
@@ -0,0 +1,69 @@
+(function ($, crmApi, crmStatus, ts, locationService) {
```

Comment:
Are you sure we wont be needing AngularJS here?

Reply:
@deb1990 we'll only use Javascript for two things:

1 - Deleting letterheads.
2 - Displaying the letterhead in the email/pdf forms.

For #1 the delete link is generated in PHP by core, so if we want to target it using Angular we'd need a directive that listen to any "delete" link. Since this is no different than using jQuery directly then there's no benefit in using Angular.

For #2 we can only use jQuery since this is a core form outside our codebase.